tests/test_sensors.py

```diff
@@ -54,6 +55,8 @@ def _render_and_load_gt(sim, scene, sensor_type, gpu2gpu):
             "{}-{}.npy".format(osp.basename(osp.splitext(scene)[0]), sensor_type),
         )
     )
+    if not osp.exists(gt_obs_file):
```

Comment:
Please add ground truth or make a separate test.  Currently this test is a half-decent integration test for the full system and I don't like making part of it just a smoke test.